src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -104,34 +104,6 @@ public function testTransWithCachingWithInvalidLocale()
         $translator->trans('foo');
     }
 
-    public function testLoadResourcesWithCaching()
```

Comment:
Don't know, but maybe you want to use the default ConfigCacheFactory in that case?